script/openqa-clone-custom-git-refspec

```diff
@@ -111,7 +111,7 @@ clone_job() {
         local json_url=${host}/tests/${job}/file/vars.json
         local json_data
         json_data=$(eval "${curl_openqa} -s ${json_url}")
-        echo "$json_data" | jq &>/dev/null || fail "openQA job with no valid JSON data encountered. Please select another job, e.g. in the same scenario: $host/t$job#next_previous"
+        echo "$json_data" | jq &>/dev/null || fail "unreadable openQA job or no valid JSON data encountered. Please try 'curl $json_url' or select another job, e.g. in the same scenario: $host/t$job#next_previous"
```

Comment:
```suggestion
        echo "$json_data" | jq &>/dev/null || fail "Unreadable openQA job or no valid JSON data encountered. Please try 'curl $json_url' or select another job, e.g. in the same scenario: $host/t$job#next_previous"
```

Also, this seems a lot more than 120 characters.

Reply:
A line break after the `||` would be nice.

Reply:
done